frontend/src/actionCreators/parcelsActionCreator.ts

```diff
@@ -81,38 +81,39 @@ export const fetchPropertyDetail = (
     : dispatch(fetchBuildingDetail({ id }, position));
 };
 
-export const createParcel = (parcel: IParcel) => (dispatch: Function) => {
+export const createParcel = (parcel: IParcel) => async (dispatch: Function) => {
   dispatch(request(actionTypes.ADD_PARCEL));
   dispatch(showLoading());
-  return CustomAxios()
-    .post(ENVIRONMENT.apiUrl + API.PARCEL_ROOT, parcel)
-    .then((response: AxiosResponse) => {
-      dispatch(success(actionTypes.ADD_PARCEL, response.status));
-      dispatch(fetchParcelDetail(response.data));
-      dispatch(hideLoading());
-    })
-    .catch((axiosError: AxiosError) => {
-      dispatch(error(actionTypes.ADD_PARCEL, axiosError?.response?.status, axiosError));
-      throw Error(axiosError.response?.data.details);
-    })
-    .finally(() => dispatch(hideLoading()));
+  try {
+    const { data, status } = await CustomAxios().post(ENVIRONMENT.apiUrl + API.PARCEL_ROOT, parcel);
```

Comment:
yeah, it would be nice to use the queryparams object that axios supports. Something like: await axios.get('https://httpbin.org/get', { params: { answer: 42 } });